ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -366,6 +367,29 @@ export default function BuildQuote({
             defaultToAll
           />
         </div>
+        {selectedToToken?.address &&
+          selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address && (
+            <div className="build-quote__token-message">
+              {t('verifyThisTokenOn', [
+                <a
+                  className="build-quote__token-etherscan-link"
+                  key="build-quote-etherscan-link"
+                  onClick={() => {
+                    const etherscanUrl = `https://etherscan.io/token/${selectedToToken.address}`
+
+                    global.platform.openTab({ url: etherscanUrl })
+                  }}
```

Comment:
The browser still doesn't know that this is being used as an anchor, because `href` isn't given. As a result, this isn't selectable via keyboard.

If instead we use `href`, it works via keyboard or mouse.

```suggestion
                  href={`https://etherscan.io/token/${selectedToToken.address}`}
                  target="_blank"
                  rel="noopener noreferrer"
```

Reply:
committed, thank-you, I'll note that for the future